source/module_io/test/support/LOWF_K_1.txt

```diff
@@ -0,0 +1,308 @@
+1 (index of k points)
```

Comment:
How large are the wave function files? It would be better to minize the size of LOWF files.